omxd.c

```diff
@@ -58,10 +96,24 @@ int main(int argc, char *argv[])
 	return 0;
 }
 
-/* Fork, umask, SID, chdir, close, logfile, FIFO */
+/* Open the FIFO at /var/run/omxctl */
+static int open_fifo(void)
+{
+	/* Create and open FIFO for command input as stdin */
+	unlink("omxctl");
+	LOG(1, "open_fifo: Deleted original omxctl FIFO.\n");
+	const mode_t old_mask = umask(0);
```

Comment:
Are we afraid here of an umask that disables writing for group and others by default? If so, very well spotted! :-)